### PR TITLE
[ticket/15745] Uses {{ CREDIT_LINE }} instead of hardcoded language

### PR DESCRIPTION
**Pull request by Crizz0**
PHPBB3-15745

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

https://tracker.phpbb.com/browse/PHPBB3-15745

### CODE REVIEW

File: phpBB/adm/style/installer_footer.html [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 	<div id="page-footer">
 		<div class="copyright">
-			Powered by <a href="https://www.phpbb.com/">phpBB</a>&reg; Forum Software &copy; phpBB Limited
+			<!-- IF S_COPYRIGHT_HTML -->
```
Comment (3D-I) [inline phpBB/adm/style/installer_footer.html]: Please use Twig as recommended.

**Comment by Crizz0** [comment]
Is this error related to my PR?

**Comment by 3D-I** [comment]
> 
> 
> Is this error related to my PR?

No.

**Comment by DavidIQ** [comment]
Just a `curl` error. Restarted that specific job.

**Pull request merged**